### ISSUE TITLE
Maybe adopt a pipe style?

### ISSUE THREAD

**Issue by zx8754** [seasmith/ggloop#2]
Would it be possible to implement *pipe* `%>%` instead of `%L+%` ?

**Comment by seasmith** [comment]
Yes, it would be possible by writing aliases for ggplot2 functions. You can look at [thomas85/pipeplotter](https://github.com/thomasp85/pipeplotter) as an example.
